geomesa-arrow/geomesa-arrow-jts/src/main/java/org/locationtech/geomesa/arrow/jts/impl/AbstractLineStringVector.java

```diff
@@ -64,8 +64,13 @@ public void set(int index, LineString geom) {
       for (int i = 0; i < geom.getNumPoints(); i++) {
         final Coordinate p = geom.getCoordinateN(i);
         tuples.setNotNull(position + i);
-        writeOrdinal((position + i) * 2, p.y);
-        writeOrdinal((position + i) * 2 + 1, p.x);
+        if (getFlipAxisOrder()) {
```

Comment:
- I would prefer to keep the internal representation of the flip axis order hidden behind the accessor (`isFlipAxisOrder()`) to encapsulate it and make it much easier to modify in one place if need be in the future because the performance trade off of looking up the method and invoking it should be negligent and will most likely be inlined by the JVM assuming that this code path is executed with enough frequency

Reply:
sure, in that case, the variable should be private and not protected.

Reply:
ah, I see you already changed that :+1: